cv32e40p/sim/ExternalRepos.mk

```diff
@@ -31,10 +31,13 @@ EMBENCH_REPO    ?= https://github.com/embench/embench-iot.git
 EMBENCH_BRANCH  ?= master
 EMBENCH_HASH    ?= 6934ddd1ff445245ee032d4258fdeb9828b72af4
 
-COMPLIANCE_REPO   ?= https://github.com/riscv/riscv-compliance
-COMPLIANCE_BRANCH ?= master
-# 2020-08-19
-COMPLIANCE_HASH   ?= c21a2e86afa3f7d4292a2dd26b759f3f29cde497
+# TODO: silabs-hfegran: Temporary fork compliance suite to support bitmanip and
```

Comment:
@silabs-hfegran, is this change to 40p deliberate?

Reply:
I believe so, changes were made to support (at the time) latest compliance suite, but bitmanip had yet to be included. This required changes to top-level core-v-verif files, and as not to break compatibility, this was added to 40p as well.